### PR TITLE
Support Gene Artwork refinements

### PR DESCRIPTION
**Pull request by orta**
This is the Eigen side of getting Gene refinement

Partnered with https://github.com/artsy/emission/pull/380

and  https://github.com/artsy/emission/pull/406

### CODE REVIEW

**Comment by orta** [comment]
This has API breaking changes for Emission - needed to change the API to include current and default settings

**Comment by orta** [comment]
Path for merging:

This version of Metaphysics needs to be deployed - https://github.com/artsy/metaphysics/pull/454
This version of Emission needs to be shipped as a beta - https://github.com/artsy/emission/pull/380
Then this can be merged.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="true">PR is classed as Work in Progress</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Needs testing on a Phone if change is non-trivial</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
  </tbody>
</table>

Time | Class | Function |
| --- | ----- | ----- |350.1ms | [AuctionViewController.swift](/artsy/eigen/tree/orta-gene_default/Artsy/View_Controllers/Auction/AuctionViewController.swift#L199) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
275.0ms | [SwiftyJSON.swift](/artsy/eigen/tree/orta-gene_default/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
186.2ms | [WebSocket.swift](/artsy/eigen/tree/orta-gene_default/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
128.5ms | [LiveAuctionLotViewController.swift](/artsy/eigen/tree/orta-gene_default/Artsy/View_Controllers/Live_Auctions/LiveAuctionLotViewController.swift#L47) | @objc override func viewDidLoad()
116.1ms | [Observable.swift](/artsy/eigen/tree/orta-gene_default/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
107.3ms | [LiveAuctionsAdminViewController.swift](/artsy/eigen/tree/orta-gene_default/Artsy/View_Controllers/Live_Auctions/LiveAuctionsAdminViewController.swift#L22) | @objc override func viewDidLoad()
106.1ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/orta-gene_default/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, salesPerson: LiveAuctionsSalesPersonType, sideMargin: String)
78.3ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/orta-gene_default/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L25) | func ar_presentBlurredOverlayWithTitle(title: String, subtitle: String, buttonState: BlurredStatusOverlayViewCloseButtonState = default)
76.1ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/orta-gene_default/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L31) | (closure)


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by alloy** [comment]
What’s the status on this?

**Comment by orta** [comment]
This build is failing due to a non-backwards compatible change in the Emission Home VC ( the removal of the init function from the public header: https://github.com/artsy/emission/commit/9cf9102fb0bf3cc6fa70e265bc8329434b591cab#diff-71ef7d5e299db20f70ef9854eb5ab4beL7

**Comment by orta** [comment]
@alloy this is good to go

File: CHANGELOG.yml [hunk 1]
```diff
@@ -39,6 +46,8 @@ releases:
     details: Second patch fix for 3.0.0 release.
     date: Nov 19, 2016
     user_facing:
+      - Refine existing queries on the Gene Page - orta
+
```
Comment (alloy) [inline CHANGELOG.yml]: Needs to move to the top.

**Comment by alloy** [comment]
Ok, resolved merge conflict, letting it do a final CI run.

@orta Otherwise, final check, this is good to go yeah?

**Comment by alloy** [comment]
```
❌  /Users/distiller/eigen/Artsy/View_Controllers/Gene/GeneRefineSettings.swift:9:157: function types cannot have argument label 'newRefineSettings'; use '_' instead
```

@orta ^

**Comment by orta** [comment]
Those fails are due to not having an updated emission deploy

**Comment by orta** [comment]
As the eigen-emission comms API had to change between the releases

**Comment by alloy** [comment]
Ahhh, makes sense. Ok, I’ll do that soon then.

**Comment by alloy** [comment]
@orta The build is still failing in the same way :/

**Comment by orta** [comment]
lols, I had been pushing my work to orta-orta-gene_refine - https://github.com/artsy/eigen/compare/orta-orta-gene_default

**Comment by orta** [comment]
screen shot 2017-02-17 at 11 30 55

**Comment by alloy** [comment]
Thanks! 🤞

**Comment by orta** [comment]
Greens locally, got a 65, will see if that gets fixed in a re-run

**Comment by orta** [comment]
it greened, looks like Danger's gonna make sure we have the build set up correctly since the release though

**Comment by alloy** [comment]
Woohoo! Quickly gonna merge before it suddenly goes red again 😄

**Pull request merged**